test/symbol-test.js

```diff
@@ -64,8 +64,8 @@ tape("symbol.type(symbolHexagonAlt) generates a polygon with the specified size"
 
 tape("symbol.type(symbolHexagonAlt) generates the expected path", function(test) {
   var s = shape.symbol().type(extra.symbolHexagonAlt).size(function(d) { return d; });
-  test.pathEqual(s(0), "M0,0L0,0L0,0L0,0L0,0L0,0L0,0Z");
-  test.pathEqual(s(10), "M1.961887,0L1.961887,0L0.980944,1.699044L-0.980944,1.699044L-1.961887,0L-0.980944,-1.699044L0.980944,-1.699044Z");
+  test.pathEqual(s(0), "M0,0L0,0L0,0L0,0L0,0L0,0Z");
+  test.pathEqual(s(10), "M1.961887,0L0.980944,1.699044L-0.980944,1.699044L-1.961887,0L-0.980944,-1.699044L0.980944,-1.699044Z");
```

Comment:
<img width="457" alt="1__vim" src="https://cloud.githubusercontent.com/assets/529247/25559108/8e0d09e4-2d3d-11e7-976d-54f9fb481600.png">
The first point was used two times again.